sources/packages/api/src/route-controllers/institution/institution.controller.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  Body,
+  Controller,
+  Get,
+  Post,
+  UnprocessableEntityException,
+} from "@nestjs/common";
+
+import { Public } from "../../auth/decorators/public.decorator";
+
+import { InstitutionService, UserService } from "../../services";
+import { CreateInstitutionDto } from "./models/institution.dto";
+import { UserToken } from "../../auth/decorators/userToken.decorator";
+import { IUserToken } from "../../auth/userToken.interface";
+import BaseController from "../BaseController";
+
+@Controller("institution")
+export class InstitutionController extends BaseController {
+  constructor(
+    private readonly userService: UserService,
+    private readonly institutionService: InstitutionService,
+  ) {
+    super();
```

Comment:
Please check if the "super" needs to be called here. I think that this could be removed.

Reply:
Done